<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Распознавание объектов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #394165;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddImageButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Добавить изображение</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
background-color: #13A6AD;
border: 1px solid white;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Resources/Resource.qrc">
      <normaloff>:/Assets/png/AddImage.png</normaloff>:/Assets/png/AddImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #ED4763;
border: 1px solid white;
</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>10.000000000000000</width>
      <height>10.000000000000000</height>
     </rectf>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveImageButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Удалить изображение</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
background-color: #13A6AD;
border: 1px solid white;
color: white;</string>
    </property>
    <property name="icon">
     <iconset resource="../Resources/Resource.qrc">
      <normaloff>:/Assets/png/RemoveImage.png</normaloff>:/Assets/png/RemoveImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 28px;
color: white;
</string>
    </property>
    <property name="text">
     <string>Название объекта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #ED4763;
border: 1px solid white;
color: white;
font-size: 28px;</string>
    </property>
    <property name="text">
     <string>фильтры</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>192</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="BlurHorizontalLayout">
     <item>
      <widget class="QCheckBox" name="BlurCheckBox">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
font-size: 28px;</string>
       </property>
       <property name="text">
        <string>Размытие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="BlurVerticalSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="sliderPosition">
        <number>3</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecognitionButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Распознать объект(ы)</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
background-color: #13A6AD;
border: 1px solid white;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Resources/Resource.qrc">
      <normaloff>:/Assets/png/FaceRecognition.png</normaloff>:/Assets/png/FaceRecognition.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="RecognitionHorizontalLayout">
     <item>
      <widget class="QRadioButton" name="FaceRecognitionRadioButton">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 16px;
background-color: #13A6AD;
border: 1px solid white;
color: white;</string>
       </property>
       <property name="text">
        <string>Распознавание лиц</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ObjectRecognitionRadioButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 16px;
background-color: #13A6AD;
border: 1px solid white;
color: white;</string>
       </property>
       <property name="text">
        <string>Распознавание объектов</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Resources/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
